data/json/mapgen/lab/lab_modular/lab_nests_modular/lab_nested_mutagen.json

```diff
@@ -0,0 +1,1033 @@
+        "q    |   ",
+        "qQQQ -   ",
+        "q    |   ",
+        "a cA X   ",
+        " Ac  X   ",
+        "JJ 6e|   "
+      ],
+      "palettes": [ "lab_common_palette" ],
+      "items": {
+        "a": [ { "item": "lab_mutagen_ingredients_fridge", "chance": 50 }, { "item": "virology_lab_fridge", "chance": 50 } ],
+        "c": [ { "item": "mut_lab", "chance": 30 } ],
+        "J": [ { "item": "mut_lab", "chance": 30 } ],
+        "Q": [ { "item": "lab_glassware", "chance": 30 } ],
+        "q": [ { "item": "lab_mutagen_ingredients", "chance": 40 } ]
+      },
+      "computers": {
+        "6": {
+          "name": "Centrifuge",
+          "options": [ { "name": "Analyze blood", "action": "blood_anal" } ],
+          "failures": [ { "action": "destroy_blood" } ]
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for a 5x9 room, includes door on west wall, possible door on south wall.",
+    "nested_mapgen_id": "lab_mutagen_9x9_W",
+    "object": {
+      "mapgensize": [ 9, 9 ],
+      "rows": [
+        "   |JJ 6e",
+        "   X Ac  ",
+        "   X  cAa",
+        "   |    q",
+        "   - QQQq",
+        "   |    q",
+        "   X Ac a",
+        "   X  cA ",
+        "   |e6 JJ"
+      ],
+      "palettes": [ "lab_common_palette" ],
+      "items": {
+        "a": [ { "item": "lab_mutagen_ingredients_fridge", "chance": 50 }, { "item": "virology_lab_fridge", "chance": 50 } ],
+        "c": [ { "item": "mut_lab", "chance": 30 } ],
+        "J": [ { "item": "mut_lab", "chance": 30 } ],
+        "Q": [ { "item": "lab_glassware", "chance": 30 } ],
+        "q": [ { "item": "lab_mutagen_ingredients", "chance": 40 } ]
+      },
+      "computers": {
+        "6": {
+          "name": "Centrifuge",
+          "options": [ { "name": "Analyze blood", "action": "blood_anal" } ],
+          "failures": [ { "action": "destroy_blood" } ]
+        }
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for a 5x9 room, includes door on east wall, possible door on south wall.",
+    "nested_mapgen_id": "lab_mutagen_9x9_E",
+    "object": {
+      "mapgensize": [ 9, 9 ],
+      "rows": [
+        " 71  |   ",
+        "  b  +   ",
+        "     |   ",
+        "|XXX||   ",
+        "JZJJY|   ",
+        " A   -   ",
+        "   5 X   ",
+        "₸ h  X   ",
+        "₸ IIy|   "
+      ],
+      "palettes": [ "lab_common_palette", "lab_mutagen_palette" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "//": "A nested map for a 5x9 room, includes door on west wall, possible door on south wall.",
+    "nested_mapgen_id": "lab_mutagen_9x9_W",
+    "object": {
+      "mapgensize": [ 9, 9 ],
+      "rows": [
+        "   |  27 ",
+        "   +  b  ",
+        "   |   o ",
+        "   ||XXX|",
+        "   |YJJJJ",
+        "   -  A  ",
+        "   X     ",
+        "   X   h₸",
+        "   |y II₸"
```

Comment:
Which symbol is for a possible door, BTW?

Reply:
This refers to the main mapgen's door placement.  There is a chance for a door to spawn on the south wall, so these nested maps preserve an open tile for access pathways.  I'll make the comment clearer.